applications/FluidDynamicsApplication/tests/compressible_navier_stokes_symbolic_generator_formulation_test.py

```diff
@@ -0,0 +1,115 @@
+import os
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnitTest
+
+from KratosMultiphysics.FluidDynamicsApplication.symbolic_generation.compressible_navier_stokes \
+    .compressible_navier_stokes_symbolic_generator import CompressibleNavierStokesSymbolicGenerator
+
+import KratosMultiphysics.FluidDynamicsApplication
+
+class CompressibleNavierStokesSymbolicGeneratorFormulationTest(KratosUnitTest.TestCase):
+    def setUp(self):
+        self.files_to_remove = []
+
+    def tearDown(self):
+        for file_name in self.files_to_remove:
+            KratosMultiphysics.kratos_utilities.DeleteFileIfExisting(file_name)
+        self.files_to_remove = []
+
+    @classmethod
+    def _FormatVector(cls, vector, fmt = "{:>.6}"):
+        """Mimics KratosMultiphysics.vector.__str__ but with a customizable format string."""
+        output = "[{}]".format(vector.Size())
+        output = "("
+        output = output + ", ".join([fmt.format(x) for x in vector])
+        output = output + ")"
```

Comment:
```suggestion
        output += "("
        output += output + ", ".join([fmt.format(x) for x in vector])
        output += output + ")"
```

I guess this is missing